Dockerfile

```diff
@@ -13,7 +13,22 @@ RUN go mod download
 # Copy the source code from the current directory to the Working Directory inside the container
 COPY . .
 
-RUN go test ./... \
-  && go build -ldflags='-s -w' -o main .
+RUN go build -ldflags='-s -w' -o main .
```

Comment:
Don't remove the test execution on build stage

Reply:
I remove this because it fails to build requiring Redis